gradle/flatpak-gradle-generator.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python3
+
+__license__ = 'MIT'
+import aiohttp
+import argparse
+import asyncio
+import json
+import hashlib
+import logging
+import re
+
+async def get_remote_sha256(url):
+    logging.info(f"started sha256({url})")
+    sha256 = hashlib.sha256()
+    async with aiohttp.ClientSession(raise_for_status=True) as http_session:
+        async with http_session.get(url) as response:
+            while True:
+                data = await response.content.read(4096)
+                if not data:
+                    break
+                sha256.update(data)
+    logging.info(f"done sha256({url})")
+    return sha256.hexdigest()
+
+async def parse_url(url):
+    ret = [{ 'type': 'file',
+            'url': url,
+            'sha256': await get_remote_sha256(url),
+            'dest': 'dependencies/flatRepo', }]
```

Comment:
`dependencies/flatRepo` directory is specific to Ghidra only:
+ https://github.com/NationalSecurityAgency/ghidra/blob/357ff97cb3e845caa91499c3c334629ee5b34c00/build.gradle#L56
+ https://github.com/NationalSecurityAgency/ghidra/blob/357ff97cb3e845caa91499c3c334629ee5b34c00/build.gradle#L70-L80

So we need to add a "dependencies path" in the script args.

Reply:
Done.